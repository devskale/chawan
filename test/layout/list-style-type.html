<!DOCTYPE HTML>
<ul type="circle">
<li value=-1>-1
<li value=0>0
<li>1
</ul>
<ol>
<li value=-1>-1
<li>0
<li>1
</ol>
<ol start=2>
<li value=-1>-1
<li>0
<li>1
</ol>
<ol start=2>
<li>2
<li>3
<li>4
</ol>
<ol start=-2147483648>
<li>2
<li>3
<li>4
</ol>
<ol start=-2147483648>
<li value=-2147483648>2
<li>3
<li>4
</ol>
