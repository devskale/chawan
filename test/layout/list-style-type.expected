
  ○ -1
  ○ 0
  ○ 1

-1. -1
 0. 0
 1. 1

-1. -1
 0. 0
 1. 1

 2. 2
 3. 3
 4. 4

 1. 2
 2. 3
 3. 4

48. 2
47. 3
46. 4
